docs/language/overview.md

```diff
@@ -1181,68 +1199,15 @@ produces
 <[(int64,string)]>
 ```
 
-### 6.12 Constants
+### 6.12 Casts
 
-Constants may be declared and bound to an identifier with a `const` statement,
-which has the form
-```
-const <id> = <expr>
-```
-This statement evaluates the expressions `<expr>` before any input is processed
-to determine the constant value assigned to the identifier `<id>`.
-This expression may not refer to `this` or any implied fields of `this`.
-
-Constants must appear at the beginning
-of a query or the beginning of a [parenthesized scope](#scopes).
-
-```mdtest-command
-echo '{diameter:1}{diameter:5}' | zq -z 'const PI = 3.14159 circumference:=2*PI*diameter' -
-```
-produces
-```mdtest-output
-{diameter:1,circumference:6.28318}
-{diameter:5,circumference:31.4159}
-```
-
-### 6.13 Type Definitions
-
-Named types may be declared and bound to an identifier with a `type` statement,
-which has the form
-```
-type <name> = <type>
-```
-This statement evaluates the type `<type>` before any input is processed
-to compute a type value assigned to the identifier `<id>`.
-
-In addition, a type statement creates a binding for the named type in
-the runtime's type system, which may be overridden by any input that
-redefines the named type.
-
-When referred to by the defined identifier, the use of this type is not
-overridden by the input data; however, type values that refer to the named
-type may be redefined.
-
-Type definitions must appear at the beginning
-of a query or the beginning of a [parenthesized scope](#scopes).
-
-```mdtest-command
-echo '{s:1}{s:10.0.0.1}' | zq -z 'type foo = {s:string} cast(this, foo)' -
-```
-produces
-```mdtest-output
-{s:"1"}(=foo)
-{s:"10.0.0.1"}(=foo)
-```
```

Comment:
Upon seeing this text seemed redundant with sections 3 & 4, I pre-cleared with @nwt that:

> `const` and `type` are statements, not expressions, so 6.12 and 6.13 should go away.